bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/GstarToTpTop/ZpTpt_Zp1500Nar_Tp1000NarLH_Ht/ZpTpt_Zp1500Nar_Tp1000NarLH_Ht_madspin_card.dat

```diff
@@ -0,0 +1,24 @@
+#************************************************************
+#*                        MadSpin                           *
+#*                                                          *
+#*    P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer * 
+#*                                                          *
+#*    Part of the MadGraph5_aMC@NLO Framework:              *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+
+set ms_dir ./madspingrid
+set max_running_process 1
+
+set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+#set BW_cut 15                # cut on how far the particle can be off-shell
```

Comment:
please follow the suggestions in https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T
(i.e. remove commented commands etc)